scripts/Credit-Card-Slips.sh

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
+#source the environment variables here so that the script runs correctly for the cron user
+source /etc/profile
 
 #make the logs dir if it doesn't already exist
 mkdir /mnt/alma/logs
 
 #change to the alma-scripts directory
 cd /mnt/alma/alma-scripts
 
-#install the dependencies and dev tools for them
-/usr/bin/python3.8 -m pipenv --python 3.8 install --dev
+#install the dependencies
+/usr/bin/python3.8 -m pipenv --python 3.8 install
 
 #run the update, which automatically only uses the current days files
-/usr/bin/python3.8 -m pipenv run llama cc-slips --source_email noreply@libraries.mit.edu --recipient_email ils-lib@mit.edu --recipient_email monoacq@mit.edu > /mnt/alma/logs/credit-card-slips.log 2>&1
+# IF its the PROD instance, send it to the prod email address
+[[ $WORKSPACE =~ .*prod* ]] && /usr/bin/python3.8 -m pipenv run llama cc-slips --source_email noreply@libraries.mit.edu --recipient_email ils-lib@mit.edu --recipient_email monoacq@mit.edu > /mnt/alma/logs/credit-card-slips.log 2>&1 || /usr/bin/python3.8 -m pipenv run llama cc-slips --source_email noreply@libraries.mit.edu --recipient_email zoto@vt.edu --recipient_email ehanson@mit.edu > /mnt/alma/logs/credit-card-slips.log 2>&1
```

Comment:
Instead of hard-coding  y'alls email addresses here, can we create a Moira list like we did for the timdex concat? Maybe something like 'alma-scripts-developers' and just use it for all of these scripts? Probably good for it to include me and Adam as well since we're all effectively maintaining this whole repo, at least for now. Easier to maintain/update as people come and go, and folks can easily set up email filters to manage the volume. Also this one I think has your old email address :)

Reply:
yeah i can update it to a list, will fix.